### ISSUE TITLE
treatment vs publication

### ISSUE THREAD

**Issue by myrmoteras** [plazi/BLR-website#59]
What is the difference between a treatment and a publication?

![image](https://user-images.githubusercontent.com/4609956/86633416-eb4cc080-bfd0-11ea-83e1-a708c1a0d672.png)

In this case, all the treatments are also publications. It seems that this is not correct, unless a publication is something published on Zenodo?

In a taxonomic and journal publishing sense, this should be separated. All the treatments should be taxon treatments, but not publications.

This way, it is impossible to find the publications by Linnaeus.

**Comment by teodorgeorgiev** [comment]
All publications are fetched from Zenodo. The "Taxonomictreatment" is a type of publication in Zenodo. If you don't want them in the publication tab, @punkish can filter them.

**Comment by punkish** [comment]
hello @myrmoteras, if you look at the [Zenodeo docs](https://zenodeo.org/docs), you will see that the **publication** param can be one of *all, article, report, book, thesis, section, workingpaper, preprint* and it defaults to *all*. As @teodorgeorgiev mentioned, *taxonomictreatment* is also a type of publication, so since the Zenodeo param **publication** defaults to *all*, it ends up fetching *taxonomictreatment*. Seems to me that you don't want that to happen so I can try and eliminate them from the grouping. Please reply to this issue with affirmative if you want me to proceed.

**Comment by myrmoteras** [comment]
Yes, I suggest we should separate treatments from publications. There is a separate tab for treatments.

**Comment by punkish** [comment]
Ok, makes sense. Will do it in a next update
